### ISSUE TITLE
Some inline images are not showing/downloading and the transparent pixel is being shown

### ISSUE THREAD

**Issue by seesemichaelj** [nylas-mail-lives/nylas-mail#63]
this is particularly noticeable when the sender is using some specific platforms (i.e. Outlook, etc). best guess is the mime type isn't being handled properly. in the meantime, the only way to see the image is to open the email in another less-desirable email client

**Comment by dweremeichik** [comment]
@seesemichaelj will you be working on this one for hacktoberfest?

**Comment by seesemichaelj** [comment]
I plan on it unless someone wants it! I can surely find a handful of stuff to fix on other repos too

**Comment by seesemichaelj** [comment]
i havent done any meaningful progress towards fixing this yet

**Comment by dweremeichik** [comment]
I was just curious in case someone else was interested. Let's let you have this one, you know the most about it. Happy hacktoberfest!

**Comment by dweremeichik** [comment]
Grave digging from Slack for posterity - transparent pixel in question: https://github.com/nylas/nylas-mail/search?utf8=%E2%9C%93&q=data%3Aimage%2Fpng%3Bbase64%2CiVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR4nGNikAQAACIAHF%2FuBd8AAAAASUVORK5CYII%3D&type=

**Comment by seesemichaelj** [comment]
Definitely going to need some help on this one! I may give up and let someone smarter figure this one out.

The file `message-item-body.cjsx` is a result of not being able to download the image; in these cases the images are part of a multipart email with inline images. It seems that they fail downloading, and therefore the cid part of the src in the html img tag doesnt get replaced by the image data, causing the transparent pixel to be displayed.

`inline-image-listeners.es6` is also after the fact from what i could tell

it appeared that `file-download-store.es6` did most of the work of downloading an image. I'm not sure if the image is corrupt or what; however outlook has no issue decoding the image. It is also 100% consistent; if an image doesnt download, it will never download (via restarting the application). ones that do download seem to always download. This points to some sort of issue with the image contents itself.

I tried uploading the image contents from the "original body" into an online base64 decoder and *did not* get the correct image. Unfortunately I'm having issues finding an image which I have the issue on nylas but can see the original source (it seems i cant see the base64 data of the image MIME type in outlook?) to see if the original is actually the original

/shrug

**Comment by seesemichaelj** [comment]
Not true! i was able to find an email that had source on gmail and didnt render on Nylas. I compared the original source on both platforms and they were the same except for the headers (slightly diffrent headers)

[original_msg.txt](https://github.com/nylas-mail-lives/nylas-mail/files/1381082/original_msg.txt)

**Comment by seesemichaelj** [comment]
Unfortunately, I don't think I have time to solve this issue before Hacktoberfest ends so others are welcome to work this!

**Comment by fannarsh** [comment]
I think this issue is related https://github.com/nylas/nylas-mail/issues/3252
And my guess is that it could have something to do with the dpi of the the image in question.
